internal/user_response.go

```diff
@@ -9,7 +9,7 @@ type UsersResponse struct {
 	// Stores additional data not described in the OpenAPI description found when deserializing. Can be used for serialization as well.
 	additionalData map[string]interface{}
 	//
-	odataNextLink *string
```

Comment:
This is inconsistent with the generated models. 
https://github.com/microsoftgraph/msgraph-sdk-go/blob/8c60e608a210cc6747dfac1ed3ae8d1ed3a7f425/models/access_package_assignment_collection_response.go#L12

Or is this an upcoming change?

Reply:
Good catch! Not that it has any impact (implementation details), but the more aligned we can be, the better.